istio.VERSION

```diff
@@ -0,0 +1,6 @@
+# Do not edit that file manually instead use
+# scripts/update_version.sh -h
+INIT = "docker.io/istio/init:2017-03-22-21.21.48"
+MIXER = "docker.io/istio/mixer:2017-03-22-19.36.02"
+RUNTIME = "docker.io/istio/runtime:2017-03-22-21.21.48"
```

Comment:
can we call this as manager? This word runtime is super confusing. People have come to terms with mixer/manager :).